internal/iface/iface.go

```diff
@@ -0,0 +1,52 @@
+package iface
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/golang/glog"
+)
+
+// DropReason is the reason why a packet was dropped.
+type DropReason int
+
+const (
```

Comment:
Constant block needs a docstring (or each constant separately, but the entire block is probably enough)